crates/sui-framework/sources/balance.move

```diff
@@ -27,6 +31,29 @@ module sui::balance {
         self.value
     }
 
+    /// Get the `Supply` value.
+    public fun supply<T>(supply: &Supply<T>): u64 {
```

Comment:
A different name here? `supply_value`? I just find the naming a little confusing, I would expect `supply` to have the signature `(&Supply): u64`